setup.py

```diff
@@ -14,7 +14,6 @@
     # zip_safe=False,
     install_requires=[
         'lmfit',
-        'fitsio'
```

Comment:
Isn't fitsio a requirement of descwl?

Reply:
whoops meant to put it back after realizing that